ui/src/components/Kubernetes/MainContent/index.vue

```diff
@@ -466,6 +474,12 @@ onBeforeUnmount(() => {
 </script>
 
 <template>
+  <SearchInput
+    v-if="showSearchInput"
+    :is-kubernetes="true"
+    :search-addon="searchAddonRef"
+    @close="showSearchInput = false"
+  />
   <n-layout :native-scrollbar="false" content-style="height: 100%" :style="themeColors">
     <n-tabs
       v-model:value="nameRef"
```

Comment:
No issues found.
The code doesn't contain any bugs. It uses Vue.js's `ref` and other components such as `UseDraggablePlus` to manipulate UI elements efficiently.

Optimization Suggestion:
1. Consider using computed property instead of ref if it is not frequently changing value because computing values takes much time when they change often like this.
2. Remove unused libraries which are not required anymore.